### PR TITLE
bucketlist link in item detail

### PR DESCRIPTION
**Pull request by rastislav-chynoransky**
https://jira.sng.sk/browse/NSNG-258

### CODE REVIEW

File: resources/js/views/ItemDetail.vue [hunk 1]
```diff
@@ -1,94 +1,122 @@
...
+            <router-link :to="{ name: 'my_collection' }" class="mt-3 block">
+                <Thumbnail class="bg-white">
+                    <template #image>
+                        <div class="flex h-full items-center justify-center bg-green">
+                            <SvgQuestion />
+                        </div>
+                    </template>
+                    <template #title>{{ $t('Hidden artworks') }}</template>
+                    <template #description
+                        >{{ found.length }}/{{ bucketlist.items.length }} collected. Tap to show the full list</template
```
Comment (igor-kamil) [inline resources/js/views/ItemDetail.vue:15]:
```suggestion
                        >{{ found.length }}/{{ bucketlist.items.length }} {{ $t('collected') }}. {{ $t('Tap to show the full list') }}</template
```

**Review by igor-kamil — APPROVED** [review]
LGTM. I am just thinking, how would this behave, when we switch the bucketlist ID to another..

**Comment by rastislav-chynoransky** [comment]
> LGTM. I am just thinking, how would this behave, when we switch the bucketlist ID to another..

You would have to search for those new items but if you've already visited them you would have them already in found artworks.

**Comment by rastislav-chynoransky** [comment]
> LGTM. I am just thinking, how would this behave, when we switch the bucketlist ID to another..

You would have to search for those new items but if you've already visited them you would have them already in found artworks.

**Pull request merged**